cpp/src/Slice/msbuild/slice.vcxproj.filters

```diff
@@ -17,381 +17,20 @@
   <ItemGroup>
     <ClCompile Include="..\FileTracker.cpp" />
     <ClCompile Include="..\Grammar.cpp" />
+    <ClCompile Include="..\MetadataValidator.cpp" />
     <ClCompile Include="..\Parser.cpp" />
     <ClCompile Include="..\Preprocessor.cpp" />
     <ClCompile Include="..\Scanner.cpp" />
     <ClCompile Include="..\SliceUtil.cpp" />
     <ClCompile Include="..\StringLiteralUtil.cpp" />
-    <ClCompile Include="..\FileTracker.cpp" />
```

Comment:
All of these files were either duplicates or references to files that no longer existed.
I'm not sure how this happened, or why they haven't been automatically removed.
So I removed them manually.